### PR TITLE
Added more detailed info to the README.md

### PR DESCRIPTION
**Pull request by Ev1lbl0w**
Fleshed out more the `README.md` info with more information:

- Added dependencies and instructions on getting Komorebi to run on some major distros
- Added optional arguments that can be used
- Added FAQ section for video perfomance tricks 
- Removed Awesome badge as AFAIK komorebi was never added to the list, and it's *kind of* shameless self-promotion :stuck_out_tongue: 
- Minor improvements overall

This is a draft for now but it is ready for review; I just need to finish up some more stuff, and wait for the `.deb` package to be finished.

Any addition/changes and opinions on the wording welcome, and it's also very likely there's a typo or two hidden around.

### CODE REVIEW

**Comment by Kangie** [comment]
In the real-world I'm a tech writer, so I'll review this over the weekend.

File: README.md [hunk 1]
```diff
@@ -23,70 +22,123 @@ It provides fully customizeable image, video, and web page wallpapers that can b
 
 ![s1](https://raw.githubusercontent.com/Komorebi-Fork/komorebi/master/screenshots/collage.jpg)
 
+## Installing
 
-## How do I install Komorebi?
+Komorebi was tested on:
```
Comment (Kangie) [inline README.md:27]: Komorebi has been tested on:

File: README.md [hunk 2]
```diff
@@ -23,70 +22,123 @@ It provides fully customizeable image, video, and web page wallpapers that can b
...
+- **Fedora** _32_
+- **Manjaro** _20.0.3_
+- **Deepin** _20_
+- **Arch Linux**
+- **Gentoo**
 
-1. Download `Komorebi` from the [Komorebi releases page](https://github.com/Komorebi-Fork/komorebi/releases).
-2. Install Komorebi using your favorite package manager (aka. double click on it)
-3. Launch Komorebi!
+### Debian and derivated (Ubuntu, Deepin, Elementary OS, Pop! OS, etc...)
```
Comment (Kangie) [inline README.md:38]: ### Debian and derivatives (...) 

I'd remove the reference to "`.deb` package coming soon" - I'd be happy to provide the package I built yesterday at this point and point users at our releases page (especially with a little more polish after work today). It's not Debian repository ready yet, but it should be enough to get us out of trouble for now. 

e.g. "Download the latest .deb package from our releases page and install it with `dpkg -i komorebi_2.2.0-1.deb' (or by double-clicking on the downloaded file."

Later on we can amend the instructions to 'apt install komorebi'

File: README.md [hunk 3]
```diff
@@ -23,70 +22,123 @@ It provides fully customizeable image, video, and web page wallpapers that can b
...
+
+```bash
+sudo dnf install meson vala gcc-c++ gtk3-devel clutter-devel clutter-gtk-devel clutter-gst3-devel webkit2gtk3-devel libgee-devel gstreamer1-libav
 ```
-sudo add-apt-repository ppa:gnome3-team/gnome3 -y
-sudo add-apt-repository ppa:vala-team -y
-sudo add-apt-repository ppa:gnome3-team/gnome3-staging -y
-sudo apt install meson valac libgtk-3-dev libgee-0.8-dev libclutter-gtk-1.0-dev libclutter-1.0-dev libwebkit2gtk-4.0-dev libclutter-gst-3.0-dev
+
+There is no `.rpm` package yet, jump to the [compilling section](#compilling); if you can package Komorebi for Fedora, go ahead and let us know!
```
Comment (Kangie) [inline README.md:54]: [compiling section](#compiling) - I'll only note this here but it's in a few places.

File: README.md [hunk 4]
```diff
@@ -23,70 +22,123 @@ It provides fully customizeable image, video, and web page wallpapers that can b
...
+and jump to the [compilling section](#compilling).
+
+## Compilling
+
+Run the following:
+```bash
 git clone https://github.com/Komorebi-Fork/komorebi.git
 cd komorebi
-meson build && cd build && meson install
+meson build && cd build && ninja
```
Comment (Kangie) [inline README.md:78]: Should we be instructing users to call Ninja, or just 'meson compile' to compile the software? 

I'll leave it up to you, the end result is identical!

File: README.md [hunk 5]
```diff
@@ -23,70 +22,123 @@ It provides fully customizeable image, video, and web page wallpapers that can b
...
 ```
 
-## Change Wallpaper & Desktop Preferences
+To install it as well, run this after the above:
```
Comment (Kangie) [inline README.md:81]: "To install the compiled package:
```
    ...
```

File: README.md [hunk 6]
```diff
@@ -23,70 +22,123 @@ It provides fully customizeable image, video, and web page wallpapers that can b
...
+```bash
+meson install
+```
+
+## Using Komorebi
+
+Simply run `komorebi`, or open your apps and look for **Komorebi**.
+
+Optional args:
```
Comment (Kangie) [inline README.md:90]: Args -> arguments

File: README.md [hunk 7]
```diff
@@ -23,70 +22,123 @@ It provides fully customizeable image, video, and web page wallpapers that can b
...
-### If you manually installed Komorebi
+```bash
+cd build
+sudo ninja uninstall
+```
 
-1. Open Terminal
-2. `cd komorebi/build`
-3. `sudo ninja uninstall`
+If you didn't compile Komorebi and install it from a pre-combiled package, uninstall it through your package manager like any other package.
```
Comment (Kangie) [inline README.md:117]: If you didn't compile Komorebi from source you can uninstall it through your package manager like any other package.

**Comment by Kangie** [comment]
Looking pretty good @Ev1lbl0w 

Given it a quick once over with some suggestions and a couple typo fixes. Happy for you to merge at your leisure once you've decided how you want to address the feedback. You'll need to bump the PR out of draft so I can approve it though!

File: README.md (hunk 2 above)
Comment (Ev1lbl0w) [reply README.md:38]: Yes, this text was here as a placeholder until the package is available. Would you like me to link to the GDrive link you posted for now, or wait until it's available on Github releases?

File: README.md (hunk 4 above)
Comment (Ev1lbl0w) [reply README.md:78]: Ah, I had no idea what was the equivalent meson command, so I went with safe route and said to use ninja. I'll change to that then.

**Comment by Ev1lbl0w** [comment]
Alright, I implemented your suggestions. I also decided to mention that we are a fork from the original project, on the description.

BTW I also created a wiki page on [Improving video wallpapers performance](https://github.com/Komorebi-Fork/komorebi/wiki/Improve-video-wallpapers-performance), if you want to review that as well.

I think all that's left is to put a proper link for the `.deb` package. I'll mark this PR as ready.

File: README.md (hunk 2 above)
Comment (Kangie) [reply README.md]: Let's link to the release page.

Once your PR is ready to go (and if you're happy for me to do so), I'll merge it immediately before tagging the 2.2.0 release & uploading the debian package. :)

Comment (Ev1lbl0w) [reply README.md]: Alright, I put a link to the releases page :wink:

**Pull request merged**